server/service/src/programs/program_event.rs

```diff
@@ -104,6 +104,7 @@ fn remove_event_stack(
     //      5---->20--|
     let mut current_end_datetime = longest.active_end_datetime;
     for mut current in previous_stack {
+        let current = &mut current.program_event_row;
```

Comment:
Nit/syntax Q: Could you destructure this in the for loop var?

Reply:
I updated the `previous_stack` Vec to match what it was before...